installation/resources/installer-config-gke-integration.yaml.tpl

```diff
@@ -79,10 +84,15 @@ data:
   global.tenantFetchers.subaccount-fetcher.secret.name: "compass-subaccount-fetcher-secret"
   global.tenantFetchers.subaccount-fetcher.secret.clientIdKey: "client-id"
   global.tenantFetchers.subaccount-fetcher.secret.clientSecretKey: "client-secret"
+  global.tenantFetchers.subaccount-fetcher.secret.oauthMode: "oauth-mtls"
+  global.tenantFetchers.subaccount-fetcher.secret.clientCertKey: "client-cert"
+  global.tenantFetchers.subaccount-fetcher.secret.clientKeyKey: "client-key"
   global.tenantFetchers.subaccount-fetcher.secret.oauthUrlKey: "url"
+  global.tenantFetchers.subaccount-fetcher.secret.skipSSLValidation: "true"
   global.tenantFetchers.subaccount-fetcher.oauth.client: "client_id"
   global.tenantFetchers.subaccount-fetcher.oauth.secret: "client_secret"
-  global.tenantFetchers.subaccount-fetcher.oauth.tokenURL: "http://compass-external-services-mock.compass-system.svc.cluster.local:8080/secured/oauth/token"
+  global.tenantFetchers.subaccount-fetcher.oauth.tokenURL: '{{ printf "https://%s.%s" .Values.global.externalServicesMock.certSecuredHost .Values.global.ingress.domainName }}'
```

Comment:
would this be resolved to the correct value? Have you tested it on local installation?

Reply:
It is tested.